staging/src/k8s.io/apiserver/pkg/storage/cacher/store_btree_test.go

```diff
@@ -137,3 +137,34 @@ func (f fakeOrderedLister) ListPrefix(prefixKey, continueKey string) []interface
 	return nil
 }
 func (f fakeOrderedLister) Count(prefixKey, continueKey string) int { return 0 }
+
+type fakeSnapshotter struct {
+	getLessOrEqual func(rv uint64) (orderedLister, bool)
+}
+
+var _ Snapshotter = (*fakeSnapshotter)(nil)
+
+// Add implements Snapshotter.
+func (f *fakeSnapshotter) Add(rv uint64, indexer orderedLister) {
```

Comment:
nit: can you please reorder the methods to reflect the order in the interface?

Reply:
Done in https://github.com/kubernetes/kubernetes/pull/130925